lib/sqlalchemy/ext/hybrid.py

```diff
@@ -207,6 +207,24 @@ def length(self, value):
     >>> i1.end
     17
 
+.. note:: When defining an setter for a hybrid property or method, the
+   expression method **must** retain the name of the original hybrid, else
+   the new hybrid with the additional state will be attached to the object
+   with the non-matching name. To use the example above::
+
+    class Interval(object):
+        # ...
+
+        @hybrid_property
```

Comment:
please use the "def length" method from the "Defining Setters" example so that it can be directly contrasted to the correct form.